### PR TITLE
chore: Cleanup status script

### PR DESCRIPTION
**Pull request by ddixit14**

### CODE REVIEW

File: generation/cleanup/repo-status.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set +e
+
+for repository in $(cat repositories.txt); do
+
```
Comment (suztomo) [inline generation/cleanup/repo-status.sh:6]: Remove unnecessary line

File: generation/cleanup/repo-status.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    release_please_url=https://raw.githubusercontent.com/googleapis/${repository}/main/.github/release-please.yml
+
+    curl ${release_please_url} --fail --silent >/dev/null
+
+    status_2=$?
+
+    if [ ${status_2} = 0 ] ; then
+          release_please="1"
+        else
```
Comment (suztomo) [inline generation/cleanup/repo-status.sh:27]: indent

File: generation/cleanup/repo-status.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    fi
+
+    release_please_url=https://raw.githubusercontent.com/googleapis/${repository}/main/.github/release-please.yml
+
+    curl ${release_please_url} --fail --silent >/dev/null
+
+    status_2=$?
+
+    if [ ${status_2} = 0 ] ; then
+          release_please="1"
```
Comment (suztomo) [inline generation/cleanup/repo-status.sh:26]: Use 2-space indent go/shell-style#indentation

File: generation/cleanup/repo-status.sh [hunk 4]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
```
Comment (suztomo) [inline generation/cleanup/repo-status.sh:2]: Add a brief description.

File: generation/cleanup/repo-status.sh [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+set +e
+
+for repository in $(cat repositories.txt); do
+
+    owlbot_file_url=https://raw.githubusercontent.com/googleapis/${repository}/main/.github/.OwlBot.yaml
```
Comment (suztomo) [inline generation/cleanup/repo-status.sh:7]: Can you use 2-space indent go/shell-style#indentation ?

**Review by suztomo — COMMENTED** [review]
Can you remove unnecessary empty lines? Use empty lines to separate steps.

**Comment by ddixit14** [comment]
@suztomo addressed all comments. Ready for review again.

File: generation/cleanup/repo-status.sh [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  release_please_url=https://raw.githubusercontent.com/googleapis/${repository}/main/.github/release-please.yml
+  curl ${release_please_url} --fail --silent >/dev/null
+  status_2=$?
+  if [ ${status_2} = 0 ]; then
+    release_please="1"
+  else
+    release_please="0"
+  fi
+
+  renovate_bot_url=$(curl -s https://raw.githubusercontent.com/googleapis/java-compute/main/renovate.json)
```
Comment (suztomo) [inline generation/cleanup/repo-status.sh:32]:
```suggestion
  renovate_bot_url=$(curl -s https://raw.githubusercontent.com/googleapis/${repository}/main/renovate.json)
```

File: generation/cleanup/repo-status.sh [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+  if grep -q "\"enabled\":\s*false" <<<"$renovate_bot_url"; then
+    renovate="0"
+  else
+    renovate="1"
+  fi
+
+  open_issues_count=$(gh issue list -R googleapis/${repository} -L 100 -s open | wc -l)
```
Comment (suztomo) [inline generation/cleanup/repo-status.sh:39]: Memo. Even after disabling issue, this command still returns 0. Good.

```
~/java-dataplex $ repository=java-accessapproval; gh issue list -R googleapis/${repository} -L 100 -s open | wc -l
the 'googleapis/java-accessapproval' repository has disabled issues
       0
```

Comment (suztomo) [reply generation/cleanup/repo-status.sh:39]: @ddixit14 Do you remember we wondered strange space when creating the CSV? I found that now.

image

**Pull request closed without merging**